src/elexmodel/distributions/GaussianModel.py

```diff
@@ -105,8 +107,10 @@ def to_aggregate(x):
                                 x[f"last_election_results_{estimand}"] / np.sum(x[f"last_election_results_{estimand}"])
                             ).to_numpy(),
                         ),
-                        "sigma_lower_bound": beta * math_utils.boot_sigma(x.lower_bounds.values, conf=(3 + alpha) / 4),
-                        "sigma_upper_bound": beta * math_utils.boot_sigma(x.upper_bounds.values, conf=(3 + alpha) / 4),
+                        "sigma_lower_bound": beta
+                        * math_utils.boot_sigma(x.lower_bounds.values, conf=(3 + alpha) / 4, winsorize=winsorize),
```

Comment:
I'd prefer if we passed winsorize as a parameter to `boot_sigma` without having to add it as a column in the dataframe

Reply:
Could you clarify where I'm adding winsorize to a dataframe?